rtsp.go

```diff
@@ -250,6 +258,7 @@ func (rc *rtspCamera) closeConnection() {
 		rc.client.Close()
 		rc.client = nil
 	}
+	rc.resetAU(nil)
```

Comment:
[q] Why reset to nil rather than an empty array of bytes like in the consumer helper?

Reply:
good question. I'll change `consumeAU` to set rc.au to `nil`